DayClock.cpp

```diff
@@ -20,7 +20,7 @@ const int ServoPin     = D6;            // Which Arduino pin is the servo connec
 
 // Servo degree positons
 // Adjust as needed:  Sun  Mon  Tue  Wed  Thu  Fri  Sat
-const int day[7] =   {20,  44,  67,  87,  110, 140, 167};
+const int day[7] =   {160,  133,  110,  87,  67, 44, 20};
```

Comment:
Your servo moves backwards from the three different brand ones that I have?  Weird.  I'd rather not merge this part of the change, since it would make the code wrong for my own units.

Reply:
Now you've got me doubting myself...  What servo do you have?  I'm going to experiment with some others, I'd like the code to reflect the most common use case.  :-P

Reply:
Yeah, my TGY-50090M runs the way yours does too.   WTF is with this servo I have installed?   Ah, that's the great thing about standards - there's so many to choose from!  

Okay, I've convinced myself.  I'm merging your change.  🙃

Reply:
Lol yeah I have some chinese "Tower Pro Micro Servo 9g SG90" servo. It's blue.